.gitattributes

```diff
@@ -0,0 +1,2 @@
+deploy/deployment             linguist-generated=true
```

Comment:
:heart: for idea

but it does not seem to work.
Seems to should be
```
deploy/deployment/*             linguist-generated
deploy/templates/crd/bases/*    linguist-generated
```
https://github.com/devfile/devworkspace-operator/compare/master...sleshchenko:static-deploy

Reply:
Why would the [github docs](https://docs.github.com/en/github/administering-a-repository/customizing-how-changed-files-appear-on-github) lie to me like that :D

Reply:
Cherry-picked your commit and squashed it.

Reply:
`linguist-generated=true` should work as well.
The cause it did not work I think missed `*`, so files inside folders are not considered as generated by default.